app/driver/settings_travis.py

```diff
@@ -15,11 +15,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.contrib.gis.db.backends.postgis',
-        'NAME': 'postgres',
-        'HOST': 'localhost',
+        'NAME': 'mub',
```

Comment:
Ugh, my bad. Addressed in https://github.com/WorldBank-Transport/DRIVER/commit/47b6561597cbc6133b180f8a3ebe5f840a1c59e7.